nips/xx.md

```diff
@@ -0,0 +1,25 @@
+NIP-XX
+======
+
+Proof of account protocol
+-------------------------------
+
+`draft` `mandatory`
+
+1. Users can prove they own some accounts by publishing an event to the relays
+`proof:<{pubkey}, {type}, {url}>`
+- `{pubkey}` is a pubkey, as a hex string
+- `{type}` is a string with a single letter, which can be:
```

Comment:
Are you sure you want to limit to 1 character identifiers? That's quite a small set size.

Reply:
You right, it's a bad idea. I think we should have a full name like "twitter" or "github"